.github/ISSUE_TEMPLATE/question.md

```diff
@@ -11,6 +11,8 @@ assignees: ''
 Hey there, you have a question? We are happy to answer. Please make sure no similar question was opened already.
 
 The following template is a suggestion how you can report an issue you run into whilst using our library. If you just want to ask a question, feel free to delete everything; just make sure you have a describing title :)
+
+Also note that there is a Telegram user group at https://t.me/pythontelegrambotgroup for questions about the library. Questions asked there might be answered quicker than here. In case you are unable to join our group due to Telegram restrictions, you can use our IRC channel at https://webchat.freenode.net/?channels=##python-telegram-bot to participate in the group.
```

Comment:
Please mind that there is also a users' Telegram group at https://t.me/pythontelegrambotgroup for questions about the library.